internal/cli/cli_test.go

```diff
@@ -34,8 +34,7 @@ func TestPrintConfiguration(t *testing.T) {
 	}
 
 	output, err := capture.StandardOutput(func() {
-		err := cli.PrintConfiguration(testConfig)
```

Comment:
Why not checking the error here?

Reply:
no error is returned

Reply:
why not? do you mean that it is always `nil`?